### DISCUSSION TITLE
Translation of the filtering of the grid

### DISCUSSION THREAD

**Discussion by rjaud49** [vikramlearning/blazorbootstrap#154] [Ideas]
Hello,

Your work on this library is awesome, thank you for it ! 
I got a suggestion about the grid component, the current labels of the filters are fixed in english, it would be nice to be able to custom them in the application used language.

Regards!

**Comment by gvreddy04** [comment]
Translation of the filtering of the grid #292

**Reply by rjaud49** [reply]
Thank you !
